pennylane/_device.py

```diff
@@ -72,10 +72,12 @@ def __init__(self, wires=1, shots=1000):
         self._wires = Wires(wires)
         self.num_wires = len(self._wires)
         self._wire_map = self.define_wire_map(self._wires)
+        self._execution = 0
         self._op_queue = None
         self._obs_queue = None
         self._parameters = None
 
+
```

Comment:
Would be good to run `black` (with a line length of 100) to remove this extra line, and also fix the corresponding CI test.

Reply:
Thanks for the reminder - I've gotten used to [pre-commits automating black](https://ljvmiranda921.github.io/notebook/2018/06/21/precommits-using-black-and-flake8/) (and flake8). Perhaps this would be a useful addition to the work flow in future?